app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
+  </dl>
+  <hr />
+  <%= render partial: 'shared/searchworks_item_information', locals: { request: request } %>
```

Comment:
I'm not a fan of passing the request around as a local. I guess we're doing this because we're dealing with a bunch of different resources with various instance variable names? Maybe we should consider loading them under the same name, e.g.:

```ruby
load_and_authorize_resource through: :exhibit, instance_name: 'request'
 ```

to make it easier to share common partials without `locals`?

(and this might be part of a larger refactor, not intended to hold up this PR)